AnnoyancesFilter/sections/popups.txt

```diff
@@ -100,6 +100,8 @@ sodapdf.com##.be-tooltip
 !
 ||timesofindia.indiatimes.com/ad_free_nudge_text.cms
 timesofindia.indiatimes.com##iframe[src^="https://timesofindia.indiatimes.com/ad_free_nudge_text.cms"]
+quirkbooks.com#$#.newsletterPrompt { display: none !important; }
+quirkbooks.com#$?#html { overflow: visible !important; }
```

Comment:
please remove `?`, and add `touch-action: auto!important;`

Reply:
It doesn't work.

Reply:
@wout-codes I mean quirkbooks.com#$?#html { overflow: visible !important; touch-action: auto!important; }